presto-raptor-legacy/src/main/java/io/prestosql/plugin/raptor/legacy/backup/HttpBackupModule.java

```diff
@@ -26,27 +26,28 @@
 
 import static io.airlift.configuration.ConfigBinder.configBinder;
 import static io.airlift.http.client.HttpClientBinder.httpClientBinder;
+import static java.util.Objects.requireNonNull;
 
 public class HttpBackupModule
         implements Module
 {
+    private final NodeManager nodeManager;
+
+    public HttpBackupModule(NodeManager nodeManager)
+    {
+        this.nodeManager = requireNonNull(nodeManager, "nodeManager is null");
+    }
+
     @Override
     public void configure(Binder binder)
     {
         configBinder(binder).bindConfig(HttpBackupConfig.class);
         binder.bind(BackupStore.class).to(HttpBackupStore.class).in(Scopes.SINGLETON);
+        binder.bind(NodeManager.class).toInstance(nodeManager);
```

Comment:
We shouldn't need to bind this here, since the instance is already bound in the anonymous module in `RaptorConnectorFactory`. This is all the same Guice instance, so it will be available in `HttpBackupStore`.

Reply:
👍  fixed.  I did this initially to fix an error with `TestHttpBackupStore` overriding this module, because it didn't have a `NodeManager` bound.  Binding a `TestingNodeManager` to the test override module works